services/core-web/common/actionCreators/permitActionCreator.js

```diff
@@ -106,6 +106,33 @@ export const createPermitAmendment = (mineGuid, permitGuid, payload) => (dispatc
     .finally(() => dispatch(hideLoading("modal")));
 };
 
+export const createPermitAmendmentVC = (mineGuid, permitGuid, permitAmdendmentGuid) => (
+  dispatch
+) => {
+  dispatch(request(reducerTypes.PERMIT_AMENDMENT_ISSUE_VC));
+  dispatch(showLoading());
+  return CustomAxios()
+    .post(
+      `${ENVIRONMENT.apiUrl}${API.PERMIT_AMENDMENT_ISSUE_VC(
+        mineGuid,
+        permitGuid,
+        permitAmdendmentGuid
+      )}`,
+      {},
+      createRequestHeader()
+    )
+    .then((response) => {
+      notification.success({
+        message: `Successfully Issued Permit Verifiable Credentials, Thanks Jason`,
+        duration: 10,
```

Comment:
lol... forgotten?

Reply:
CaN yOu PlEaSe ReMoVe ThAt PaRt 😄

Reply:
I don't know what the issue is, can you provide an example of something you would prefer to see?

Reply:
Remove the part that says `Successfully Issued Permit Verifiable Credentials, ` then I think we're good

Reply:
Please update to "Successfully Issued Permit Verifiable Credentials, Thanks Oh Great Ryan, sole developer of MDS"